src/shogun/mathematics/linalg/ratapprox/logdet/LogDetEstimator.cpp

```diff
@@ -124,39 +123,26 @@ SGVector<float64_t> CLogDetEstimator::sample(index_t num_estimates)
 		m_trace_sampler->get_dimension());
 
 	index_t num_trace_samples=m_trace_sampler->get_num_samples();
-	index_t idx_row = 0;
-	index_t idx_col = 0;
 	SGVector<float64_t> samples(num_estimates);
 	samples.zero();
-	// for omp
-#pragma omp parallel for
-		for (index_t i = 0; i < num_estimates; i++)
+	// TO DO: use openmp like this here->#pragma omp parallel for reduction(+:
```

Comment:
why is it a todo if it is the solution? if not then just add the todo but not the whole pragma.. have you tested the pragma itself?

Reply:
the pragma didnt work.... ```error: 'samples' does not have pointer or array type```